recipe/install_ucx.sh

```diff
@@ -7,6 +7,12 @@ export CONDA_BUILD_SYSROOT="$(${CC} --print-sysroot)"
 export CFLAGS="${CFLAGS} -I${CONDA_BUILD_SYSROOT}/usr/include"
 export LDFLAGS="${LDFLAGS} -L${CONDA_BUILD_SYSROOT}/usr/lib64"
 
+CUDA_CONFIG_ARG=""
+if [ ${cuda_compiler_version} != "" ]; then
+    export CFLAGS="${CFLAGS} -I${CUDA_HOME}/include"
```

Comment:
Can we do these with the cuda compiler activation scripts?

Reply:
Good point. Yeah actually I think [we are]( https://github.com/conda-forge/nvcc-feedstock/blob/ac8304a8d12c0dd9705b34d6e4d53d43fa5cada5/recipe/install_nvcc.sh#L34-L36 ). This code precedes that change. Agree it would be good to get rid of this though.

Reply:
This is now dropped. Things seem to work fine without it. Thanks for catching this 🙂